src/header/utils.test.js

```diff
@@ -11,11 +12,18 @@ const props = {
 describe('header utils', () => {
   describe('getContentMenuItems', () => {
     it('should include Video Uploads option', () => {
+      setConfig({
+        ...getConfig(),
+        ENABLE_VIDEO_UPLOAD_PAGE_LINK_IN_CONTENT_DROPDOWN: 'true',
+      });
```

Comment:
Note to reviewer: even though `.env.test` sets this to 'true' already, this test suite doesn't actually run `initialize`, so the configuration needs to be set manually.